custom_components/reflex_clerk_api/base.py

```
@@ -68,9 +69,9 @@ class ClerkBase(rx.Component):
     # The React library to wrap.
     # `Show` is exported from `@clerk/react` (v6+), not `@clerk/clerk-react`.
     library = CLERK_REACT_LIBRARY
-    lib_dependencies = list(DEFAULT_CLERK_FRONTEND_VERSIONS.dependency_libraries)
+    lib_dependencies: tuple[str, ...] = DEFAULT_CLERK_FRONTEND_VERSIONS.dependency_libraries
 
-    def __init_subclass__(cls, **kwargs):
+    def __init_subclass__(cls, **kwargs: Any) -> None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)